Source/WebCore/platform/graphics/texmap/BitmapTexturePool.cpp

```diff
@@ -50,9 +50,10 @@ RefPtr<BitmapTexture> BitmapTexturePool::acquireTexture(const IntSize& size, con
         });
 
     if (selectedEntry == m_textures.end()) {
-        m_textures.append(Entry(BitmapTexture::create(flags)));
+        m_textures.append(Entry(BitmapTexture::create(size, flags)));
         selectedEntry = &m_textures.last();
-    }
+    } else
+        selectedEntry->m_texture->reset(size, flags);
 
```

Comment:
I think you should call `m_texture->reset(size, flags);` unconditionary  to clear `m_filterInfo`.

Reply:
That's what I'm doing, no? when not found in the pool we create a new one, so filter info is empty already, and when reusing a texture reset() is called.

Reply:
oh, my bad. you are right.